account_invoice_inter_company/tests/test_inter_company_invoice.py

```diff
@@ -3,36 +3,60 @@
 # Copyright 2020 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import _
-from odoo.exceptions import UserError, ValidationError
+from unittest.mock import patch
+
+from odoo.exceptions import UserError
 from odoo.tests import tagged
 from odoo.tests.common import Form, TransactionCase
 
+from odoo.addons.account.models.chart_template import AccountChartTemplate
+from odoo.addons.account.tests.test_chart_template import test_get_data
+
+
+def _get_chart_template_mapping(self, get_all=False):
+    return {
+        "test": {
+            "name": "test",
+            "country_id": None,
+            "country_code": None,
+            "modules": ["account"],
+            "parent": None,
+        }
+    }
+
 
 @tagged("post_install", "-at_install")
+@patch.object(
+    AccountChartTemplate, "_get_chart_template_mapping", _get_chart_template_mapping
+)
 class TestAccountInvoiceInterCompanyBase(TransactionCase):
     @classmethod
+    @patch.object(
+        AccountChartTemplate, "_get_chart_template_mapping", _get_chart_template_mapping
+    )
     def setUpClass(cls):
         super().setUpClass()
         cls.account_obj = cls.env["account.account"]
         cls.account_move_obj = cls.env["account.move"]
-        cls.chart = cls.env["account.chart.template"].search([], limit=1)
-        if not cls.chart:
-            raise ValidationError(
-                # translation to avoid pylint warnings
-                _("No Chart of Account Template has been defined !")
-            )
 
         cls.company_a = cls.env["res.company"].create(
             {
                 "name": "Company A",
                 "currency_id": cls.env.ref("base.EUR").id,
                 "country_id": cls.env.ref("base.fr").id,
-                "parent_id": cls.env.ref("base.main_company").id,
+                # "parent_id": cls.env.ref("base.main_company").id,
```

Comment:
Please remove these commented lines instead of commenting them (there are several).

Reply:
Done